terraform/deploy/modules.tf

```diff
@@ -74,6 +74,10 @@ module "concourse_web" {
     https_proxy = "http://${module.vpc.outputs.internet_proxy_endpoint}:3128"
     no_proxy    = "169.254.169.254,${join(",", formatlist("%s.%s", local.endpoint_services, local.amazon_region_domain))},${local.enterprise_github_url}"
   }
+  enterprise_github_certs = [
```

Comment:
Seems like the list should be defined once and passed into both modules

Reply:
Yeah, good point. I got a bit copy-paste happy. Maybe Chris can fix up in the morning. Thanks for the prompt review at a ridiculous hour!

Reply:
Fixed this and no_proxy